### PR TITLE
Update UDP ports used by new aioswitcher version

### PR DESCRIPTION
**Pull request by regevbr**
## Proposed change

Update UDP ports used by new aioswitcher version [3.3.0](https://github.com/TomerFi/aioswitcher/releases/tag/3.3.0)

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  home-assistant/core#91215
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by thecode — APPROVED** [review]

File: source/_integrations/switcher_kis.markdown [hunk 1]
```diff
@@ -88,5 +88,5 @@ Note: This does not affect the auto off timer.
 
 ## Notes
 
-Make sure that Home Assistant host's firewall allows incoming traffic on UDP ports 20002 & 20003 and outgoing connections to Switcher device(s) on TCP ports 9957 & 10000.
+Make sure that Home Assistant host's firewall allows incoming traffic on UDP ports 10002 & 20002 & 20003 and outgoing connections to Switcher device(s) on TCP ports 9957 & 10000.
```
Comment (thecode) [inline source/_integrations/switcher_kis.markdown]:
```suggestion
Make sure that Home Assistant host's firewall allows incoming traffic on UDP ports 10002, 20002 & 20003 and outgoing connections to Switcher device(s) on TCP ports 9957 & 10000.
```

**Review by thecode — CHANGES_REQUESTED** [review]
Just noticed we should use `,`

File: source/_integrations/switcher_kis.markdown (hunk 1 above)
Comment (regevbr) [reply source/_integrations/switcher_kis.markdown]: oh my bad

Comment (regevbr) [reply source/_integrations/switcher_kis.markdown]: done

**Review by thecode — APPROVED** [review]

**Pull request merged**